### ISSUE TITLE
AddtoCart is Not Working..

### ISSUE THREAD

**Issue by rockstarvibu** [RayOkaah/woocommerce#2]
Its really an awesome plugin but add to cart is not working.. im getting this error :- 

```
Exception has occurred.
_Exception (Exception: WooCommerce Error!
code: woocommerce_rest_missing_nonce
message: Missing the X-WC-Store-API-Nonce header. This endpoint requires a valid nonce.
status: 401)
```
and make your documentation well... its not clear how to use this... Thanks

**Comment by RayOkaah** [comment]
Thank you. This was a known issue and  fixed in the most recent version 0.9.5. I am also considering writing detailed articles, once I make time and would appreciate such from the community as well.

**Comment by rockstarvibu** [comment]
You will be appreciated once you have detailed information. because it makes clear how to use and features and all that stuff.. good luck👍
waiting for documentation eagerly.
